e2e/testmain_test.go

```diff
@@ -77,7 +78,8 @@ func TestMain(m *testing.M) {
 	if err != nil {
 		log.Fatal(err)
 	}
-	svc, err := cloud.NewService(ctx, client, &cloud.SingleProjectRouter{ID: testFlags.project}, &cloud.NopRateLimiter{})
+	mrl := &cloud.MinimumRateLimiter{RateLimiter: &cloud.NopRateLimiter{}, Minimum: 50 * time.Millisecond}
```

Comment:
why not pass this as a flag with default value 50ms? This value might change in future and this way we can configure it per project.

Reply:
1. This is mitigation and not a fix. We are still investigating the root cause
2. The value matches default quota and is still low enough to not impact test speed negatively (+7.5s is smaller than +-20s variance)